gpytorch/variational/cholesky_variational_distribution.py

```diff
@@ -47,33 +47,16 @@ def variational_distribution(self):
         that the diagonal remains positive.
         """
         chol_variational_covar = self.chol_variational_covar
+        dtype = chol_variational_covar.dtype
+        device = chol_variational_covar.device
+        mask = torch.ones(self.chol_variational_covar.shape[-2:], dtype=dtype, device=device).triu()
+        chol_variational_covar = chol_variational_covar.mul(mask)
```

Comment:
I assume that this happened because `triu()` doesn’t operate in batch mode?

Reply:
Yeah